remo/remozilla/tasks.py

```diff
@@ -41,8 +41,13 @@ def parse_bugzilla_time(time):
 
 
 @task
-@transaction.commit_on_success
 def fetch_bugs(components=COMPONENTS, days=None):
+    return
```

Comment:
Please add a comment here explaining the transition steps from cron to periodic task.